src/redux/reducers/billReducers.js

```diff
@@ -18,6 +18,17 @@ const billReducers = (state = INTIAL_STATE, action) => {
         ...state,
         bills: action.bills,
       };
+    case ACTIONS.CREATE_BILL:
```

Comment:
not needed with the actions changes as discussed within meetings

Reply:
What was the reason again? forgot